k8s-deploy/pkg/job/cluster_install.go

```diff
@@ -219,6 +219,11 @@ func createCluster(job *modules.Job) (*modules.Cluster, error) {
 		return nil, err
 	}
 
+	versionCorrect := validateVersion(cluster.ChartVersion, cluster.Spec["imageTag"].(string))
+	if !versionCorrect {
+		return nil, fmt.Errorf("the image tag is not consistent with the chart verison, which is unsupported")
+	}
```

Comment:
> How about return error directly from `validateVersion`, instead of return true or false and return error in the caller?
> 
> Also maybe we want to have the chartVersion and image tag printed in the error message?

Sure, fixed.